src/js/components/validateData/validateValues/ValidateValuesErrorReport.jsx

```diff
@@ -66,7 +66,7 @@ export default class ValidateValuesErrorReport extends React.Component {
 
             let description = item.error_description;
             if (item.error_name == 'rule_failed') {
-                description = item.rule_failed;
+                description = ['Rule ' + item.original_label + ': ' + item.rule_failed];
```

Comment:
Please look at the next line (https://github.com/fedspendingtransparency/data-act-broker-web-app/blob/665a2a4c34f9aadb2b9d2872cac5a11cb6c005c4/src/js/components/validateData/validateValues/ValidateValuesErrorReport.jsx#L68-L72). 

I followed example of the cross-file validation (https://github.com/fedspendingtransparency/data-act-broker-web-app/blob/dev/src/js/components/crossFile/components/ComparisonTable.jsx#L27)